app/shared/video-player/src/desktopMain/kotlin/ui/VideoPlayer.desktop.kt

```diff
@@ -156,6 +158,8 @@ class VlcjVideoPlayerState(parentCoroutineContext: CoroutineContext) : PlayerSta
         }
     }
 
+    override val chapters: StateFlow<MutableList<Chapter>> = MutableStateFlow(mutableListOf())
```

Comment:
```suggestion
    override val chapters: StateFlow<MutableList<Chapter>> = MutableStateFlow(emptyList())
```
emptyList 是单例, 性能更好, 虽然这里没太大区别